pna.p4

```diff
@@ -793,13 +797,13 @@ extern void update_expire_info(
 //                   if the data and value parameters are equal.
 //
 // Examples:
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_SYN &&
-//                    meta.direction == OUTBOUND,
-//                    tcp_connection_start_time_profile_id);
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_ACK,
-//                    tcp_connection_continuation_time_protile_id);
-// set_expire_time_if(hdr.tcp.flags == TCP_FLG_FIN,
-//                    tcp_connection_close_time_profile_id);
+// set_entry_expire_time_if(hdr.tcp.flags == TCP_FLG_SYN &&
```

Comment:
Were these changes intentional?

Reply:
Yes.  The name set_expire_time_if in the comments is not defined in the pna.p4 header file.  set_entry_expire_time_if is defined.